src/jvm/test/resources/expected_logs/inline_ext_fun.txt

```diff
@@ -22,7 +22,7 @@ Detailed trace:
 |   MethodHandle.linkToStatic("SOME", MemberName#1): "SOME" at DirectMethodHandle.Holder.invokeStatic(DirectMethodHandle$Holder:0)                   |
 |   DirectMethodHandle.internalMemberName(DirectMethodHandle#2): MemberName#2 at DirectMethodHandle.Holder.invokeStatic(DirectMethodHandle$Holder:0) |
 |   MethodHandle.linkToStatic(0, "SOME", MemberName#2): 4 at DirectMethodHandle.Holder.invokeStatic(DirectMethodHandle$Holder:0)                     |
-|   argL0.invokeBasic(4, 1): 5 at DelegatingMethodHandle.Holder.reinvoke_L(DelegatingMethodHandle$Holder:0)                                          |
+|   DirectMethodHandle#3.invokeBasic(4, 1): 5 at DelegatingMethodHandle.Holder.reinvoke_L(DelegatingMethodHandle$Holder:0)                           |
```

Comment:
Could you please recheck this change?

Reply:
I think this is a code generated by the Kotlin compiler to compile string template (via invokedynamic), so it's hard to say what should be the expected trace here. In any case, names `argL0` are typically auto-generated names of fields of lambda objects storing captured values, so I guess previously it was incorrectly displayed in the trace.

Reply:
We should not show all these details, let's raise an issue.

Reply:
https://youtrack.jetbrains.com/issue/DR-302/Hide-invokedynamic-and-MethodHandles-implementation-details-from-the-trace